classes/controller/FrontController.php

```diff
@@ -1217,7 +1217,7 @@ public function addJqueryUI($component, $theme = 'base', $check_dependencies = t
 
         $this->registerStylesheet('jquery-ui-theme', $css_theme_path, ['media' => 'all', 'priority' => 95]);
         $this->registerStylesheet('jquery-ui', $css_path, ['media' => 'all', 'priority' => 90]);
-        $this->registerJavascript('jquery-ui', $js_path, ['position' => 'bottom', 'priority' => 90]);
+        $this->registerJavascript('jquery-ui', $js_path, ['position' => 'bottom', 'priority' => 49]);
```

Comment:
What's the reason behind 49? :thinking: 
Why not 50, or 40?

Reply:
Looks like 49 is right before the theme.js, @Progi1984 confirm it

Reply:
Maybe add a comment, not sure we will retrieve this information in the future, it's not obvious =)